src/i18n.ts

```diff
@@ -32,4 +32,12 @@ i18n
     },
   })
 
+i18n.on('languageChanged', lang => {
+  numeral.locale(langs[lang].numeralLocale)
+})
+
```

Comment:
🆒

Reply:
> It looks like the formatting is always adding 3 fractional digits:
> ![Screen Shot 2020-03-26 at 3 56 09 PM](https://user-images.githubusercontent.com/6854428/77704552-91bd0a80-6f7a-11ea-8522-0c53d2fa68e4.png)

Yes it is, we can easily change it to be more or less, depending what we want, it's controlled by the format string in numberFormatter.

'0.000a' is the default non-rounded, humanized string, with .000 being 3 decimal places.

Reply:
i can have a lot and see if there's a way of trimming extra 0s.

Reply:
Ah yes, there is.  `[0000]`

Reply:
![image](https://user-images.githubusercontent.com/2041260/77704861-5cfd8300-6f7b-11ea-9925-0c12bf9d0295.png)